arangod/Aql/IndexNode.cpp

```diff
@@ -698,13 +652,17 @@ void IndexNode::setLateMaterialized(aql::Variable const* docIdVariable,
                                     IndexId commonIndexId,
                                     IndexVarsInfo const& indexVariables) {
   _outNonMaterializedDocId = docIdVariable;
-  _outNonMaterializedIndVars.first = commonIndexId;
-  _outNonMaterializedIndVars.second.clear();
-  _outNonMaterializedIndVars.second.reserve(indexVariables.size());
+  IndexValuesVars oldIndexValuesVars;
+  oldIndexValuesVars.first = commonIndexId;
+  oldIndexValuesVars.second.clear();
```

Comment:
`.second()` should already be empty here. Clearing it does not harm, but also does not help.
```suggestion
```